Source/ASDisplayNode.mm

```diff
@@ -3179,7 +3211,10 @@ - (ASEventLog *)eventLog
   if (self.debugName.length > 0) {
     [result addObject:@{ @"debugName" : ASStringWithQuotesIfMultiword(self.debugName)}];
   }
-  
+  if (self.accessibilityIdentifier.length > 0) {
+    [result addObject:@{ @"axId": ASStringWithQuotesIfMultiword(self.accessibilityIdentifier) }];
```

Comment:
Why does this use axId but above, the axId is set to kCFNull? Worth a comment in the kCFNull case as to why that is the key for the dictionary.